include/RGBPatternManager.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include <nonstd/span.hpp>
+
+#include <freertos/FreeRTOS.h>
+#include <freertos/semphr.h>
+#include <freertos/task.h>
+
+#include <cstdint>
+
+#include <esp32-hal-rmt.h>
```

Comment:
Preferably have this above the <freertos/....h> includes

Reply:
This is autoformatting, not mine.